src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Participant.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
```

Comment:
Undo changes in this file, line 2, 46 and 47. OK to keep the reformatting of constrctor